packages/cli/src/lib.rs

```diff
@@ -279,6 +286,44 @@ where
     // Render the template
     template.render(&target_dir, pkg_manager, &package_name, alpha, mobile)?;
 
+    // Install dependencies
+
+    if install_deps {
+        if pkg_manager.is_node() {
+            if cfg!(target_os = "windows") {
```

Comment:
use a `#[cfg]` instead, for example:
```rs
#[cfg(target_os = "windows")
{
	// windows code here
}
#[cfg(not(target_os = "windows"))
{
	// non-windows code here
}
```
you could also simplify this a bit further by replacing `target_os = "windows"` with just `windows`